hack/e2e/values.venafi-kubernetes-agent.yaml

```diff
@@ -10,3 +10,4 @@ authentication:
 
 extraArgs:
 - --logging-format=json
+- --log-level=2
```

Comment:
I changed this so that could see trace level logs in the e2e test.
As it happens, the single controller-runtime log message from venafi-connection-lib is at level 0, so this change is not strictly necessary.